packages/playground/wordpress-builds/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM emscripten/emsdk:3.1.24 as emscripten
+FROM emscripten/emsdk:3.1.61 as emscripten
```

Comment:
Should we use `4.0.5` here to match the version used by the [base-image Dockerfile](https://github.com/Automattic/wordpress-playground-private/blob/b38db761632d860bd519021d33a2230117841ee4/packages/php-wasm/compile/base-image/Dockerfile#L53).

Reply:
@bgrgicak AFAICT, v3.1.61 is [the latest Emscripten version used in this repo](https://github.com/WordPress/wordpress-playground/blob/a6d4849d2af35eae598e80b3777eaddfe5fcc09c/packages/php-wasm/compile/base-image/Dockerfile#L53).

@adamziel I figured there was some reason we were grabbing that image but didn't know why. It seems like using an ubuntu image would be fine too.